diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/DiktatRuleSetProvider.kt

```diff
@@ -22,33 +20,40 @@ import org.slf4j.LoggerFactory
  */
 const val DIKTAT_RULE_SET_ID = "diktat-ruleset"
 
-class RuleSetDiktat(val rulesConfig: List<RulesConfig>, vararg rules: Rule) : RuleSet(DIKTAT_RULE_SET_ID, *rules)
-
-fun KtLint.Params.getDiktatConfigRules(): List<RulesConfig> = (this.ruleSets.find { it.id == DIKTAT_RULE_SET_ID } as RuleSetDiktat).rulesConfig
-
 class DiktatRuleSetProvider(private val jsonRulesConfig: String = "rules-config.json") : RuleSetProvider {
     override fun get(): RuleSet {
         log.debug("Will run $DIKTAT_RULE_SET_ID with $jsonRulesConfig (it can be placed to the run directory or the default file from resources will be used)")
-        return RuleSetDiktat(
-                RulesConfigReader(javaClass.classLoader).readResource(jsonRulesConfig) ?: listOf(),
-                CommentsRule(),
-                KdocComments(),
-                KdocMethods(),
-                KdocFormatting(),
-                FileNaming(),
-                PackageNaming(),
-                FileSize(),
-                IdentifierNaming(),
-                BracesInConditionalsAndLoopsRule(),
-                BlockStructureBraces(),
-                EmptyBlock(),
-                LineLength(),
-                BlankLinesRule(),
-                WhiteSpaceRule(),
-                WhenMustHaveElseRule(),
-                FileStructureRule(),  // this rule should be right before indentation because it should operate on already valid code
-                NewlinesRule(),  // newlines need to be inserted right before fixing indentation
-                IndentationRule()  // indentation rule should be the last because it fixes formatting after all the changes done by previous rules
+        val configRules = RulesConfigReader(javaClass.classLoader).readResource(jsonRulesConfig) ?: listOf()
+        return RuleSet(
+                DIKTAT_RULE_SET_ID,
+                CommentsRule(configRules),
```

Comment:
please write a small note here that we know that it is ugly, but this is the only way for us to fix it